docs/buildah-import.md

```diff
@@ -0,0 +1,70 @@
+## buildah-import "1" "July 2017" "buildah"
+# NAME
+buildah-import - Create an empty filesystem image and import the contents of the tar archive (.tar, .tar.gz, .tgz, .bzip, .tar.xz, .txz) into it, then optionally tag it.
+
+# SYNOPSIS
+**buildah** **import** [*options* [...]] file|URL|**-**[REPOSITORY[:TAG]]
+
+# OPTIONS
+**-c**, **--change**=[]
+   Apply specified Dockerfile instructions while importing the image
+   Supported Dockerfile instructions: `CMD`|`ENTRYPOINT`|`ENV`|`EXPOSE`|`ONBUILD`|`USER`|`VOLUME`|`WORKDIR`
+
```

Comment:
Looks like you're supporting:
ARCH, CMD, CREATED-BY, ENTRYPOINT, ENV, EXPOSE, LABEL, MAINTAINTER, OS, USER, VOLUME, WORKDIR in the code, IDK if you wanted to keep a few on the down low or not.